requirements-tests.txt

```diff
@@ -0,0 +1,6 @@
+pytest==3.9.3
+pytest-cov==2.6.0
+pylint==1.9.3; python_version < "3"
```

Comment:
we can stop supporting python 2.7

Reply:
https://github.com/ClearcodeHQ/jira_timemachine/issues/11